### PR TITLE
Find transforms in package.json

### PR DESCRIPTION
**Pull request by cjhowe7**
This is how I think we can pull the `transform` field from `package.json` to merge it with the options to `sheetify`. It's a modified version of [this](https://github.com/stackhtml/documentify/blob/master/index.js#L119-L143). However, I'm not really sure how to write the tests for it. I'm going to write an example that uses this, but we'll have to figure out how to put that into the test suite.

### CODE REVIEW

**Comment by cjhowe7** [comment]
Note that the failing tests for this PR are due to `postcss-prefix@2.1.0` being pulled in. #135 will also fail if we run the Travis tests agains.

**Comment by cjhowe7** [comment]
This should be fixed by https://github.com/stackcss/postcss-prefix/pull/13

**Comment by yoshuawuyts** [comment]
Merged the linked PR - re-running tests to see if they pass

**Comment by cjhowe7** [comment]
Indeed, it passed! Would you like me to squash the commits on this pr?

https://travis-ci.org/stackcss/sheetify/builds/281125448

File: transform.js [hunk 1]
```diff
@@ -54,19 +54,25 @@ function transform (filename, options) {
...
-
-    // transform all detected nodes and
-    // close stream when done
-    mapLimit(nodes, Infinity, iterate, function (err) {
+    findTransforms(filename, function (err, transforms) {
       if (err) return self.emit('error', err)
-      self.push(ast.toString())
-      self.push(null)
+
+      opts.transform = transforms.concat(opts.transform || []).concat(opts.t || [])
```
Comment (cjhowe7) [inline transform.js]: Ehhh, I think this is wrong. Maybe if there are multiple files it will fail? Let me write a test.

**Comment by cjhowe7** [comment]
Not really sure there's much I can do about codecov complaining about patch test coverage. The one line that is not covered in the patch wasn't covered before the patch either. I guess the indentation change made codecov think it was a new line of code? Not really sure how to cause falafel to fail, either.

https://codecov.io/gh/stackcss/sheetify/commit/a002150a17aed9b487958c23cf88c6eaf346f87b#D5-66

File: transform.js (hunk 1 above)
Comment (cjhowe7) [reply transform.js]: Ah, no, that's definitely wrong. This is only called once per file, because this is in the end handler. My bad.

**Comment by cjhowe7** [comment]
👍

**Pull request merged**